packages/hardhat-ovm/test/index.ts

```diff
@@ -0,0 +1,79 @@
+import path from 'path'
+import * as fsExtra from 'fs-extra'
+import { assert } from 'chai'
+import { resetHardhatContext } from 'hardhat/plugins-testing'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+
+const hardhatProjectPath = path.join(__dirname, 'hardhat-project')
+
+describe('Optimism plugin', () => {
+  let hre: HardhatRuntimeEnvironment
+
+  before('set environment to the hardhat-project folder', async () => {
+    process.chdir(hardhatProjectPath)
```

Comment:
This is very similar to what we do in our plugin tests, except we do it in a `beforeEach` hook. We also have a `fixture-projects` directory where we have a directory for each test project. You can check it [here](https://github.com/nomiclabs/hardhat/blob/d4d9e5e28bcf6c65f2b03bcd765fc7a0da04366c/packages/hardhat-ethers/test/helpers.ts#L14)